src/app/programs/frc/robots/page.tsx

```diff
@@ -0,0 +1,171 @@
+import { Carousel } from "@/components/Carousel";
+import { Image } from "@/components/Image";
+import { ManualCarousel } from "@/components/ManualCarousel";
+import { PatternBackground } from "@/components/PatternBackground";
+import {
+  CarouselNext,
+  CarouselPrevious,
+} from "@/components/shadcn/ui/carousel";
+
+import troubleClef from "@/image/biohazard/trouble-logo.webp";
+import viperLogo from "@/image/biohazard/viper-logo.webp";
+
+import { ParallaxImage } from "../components";
+
+function SlideButtons() {
+  return (
+    <>
+      <CarouselPrevious
+        className={"w-20 h-24 opacity-85"}
+        variant={"default"}
+        size={"lg"}
+      ></CarouselPrevious>
+      <CarouselNext
+        className={"w-20 h-24 opacity-85"}
+        variant={"default"}
+        size={"lg"}
+      ></CarouselNext>
+    </>
+  );
+}
+
+const slides = [
```

Comment:
TypeScript is very smart, so it inferred that this array was of type `JSX.Element[]`. That said, I tend to lean towards defining const/variable/function types where possible. 

In this case, we could add a `string` to this array and TypeScript wouldn't complain - which probably isn't desired.

Easy fix:
```suggestion
const slides: JSX.Element[] = [
```